packages/tools/client-debugger/client-debugger-view/src/components/TelemetryView.tsx

```diff
@@ -20,9 +19,8 @@ import {
 	TableColumnDefinition,
 	createTableColumn,
 } from "@fluentui/react-components";
-import { Info24Regular, Info24Filled } from "@fluentui/react-icons";
-import React, { useState } from "react";
-
+import React, { useState, useRef } from "react";
```

Comment:
Nit: (and I'm probably guilty of introducing this at some point) instead of importing the whole `React` _and_ specific functions, let's replace the use of the functions below with `React.useState()` and `React.useRef`.